docs/getting-started/continuous_integration.md

```diff
@@ -0,0 +1,107 @@
+---
+layout: default
+title: Continuous Integration
+parent: Getting started
+nav_order: 5
+permalink: /getting-started/continuous-integration/
+---
+
+# Continuous Integration
+
+OSS-Fuzz offers Continuous Integration(CI) support for projects hosted on
+GitHub. This service is called CIFuzz and can be used to run OSS-Fuzz fuzz targets
+on your project when a pull request is submitted. It has the advantage of
+detecting bugs before they are committed into your project. This insures that
+less bugs are committed meaning less active security vulnerabilities.
+
+## How it works
+
+CIFuzz works by checking out a repository at the head of a pull request. The
+project's fuzz targets are built and run for
+a definite amount of time (default is 10 minutes). If a bug is found, the
+stack trace as well as the test case are surfaced to the user. If a bug is
```

Comment:
"are made available for download" instead of "surfaced to the user"

Reply:
Done.